magnolia-voting-app/src/main/java/com/example/votingapp/valueprovider/PollStatusValueProvider.java

```diff
@@ -0,0 +1,50 @@
+package com.example.votingapp.valueprovider;
+
+import java.util.*;
+
+import javax.jcr.*;
+
+import com.vaadin.data.*;
+
+import org.slf4j.*;
+
+/**
+ * PollStatusColumnDefinition
+ */
+public class PollStatusValueProvider implements ValueProvider<Node, String> {
+    
+    private static final Logger LOGGER = LoggerFactory.getLogger(PollStatusValueProvider.class);
+
+    public static final String TO_DO = "To-do";
+    public static final String ON_GOING = "On-going";
+    public static final String DONE = "Done";
+
```

Comment:
You only apply the variables in this class. Should use 'private' for them

Reply:
Because I think these variables are used for a general purpose and should be reused in a class of the FA-13 or any classes in the future, I made them public static.

Reply:
One class has one target, this class is for changing the status poll, if you want to reuse the variables, should create them in a common class as Util

Reply:
Yes. I have created the PollStatusUtils class for reuse purposes.